### ISSUE TITLE
MSRV of 1.58.0 is wrong?

### ISSUE THREAD

**Issue by nwalfield** [randombit/botan-rs#85]
The README says:

> The current version requires Botan 2.8.0 or higher and Rust 1.58.0 or higher.

but when compiling [sq](https://gitlab.com/sequoia-pgp/sequoia-sq) with rustc 1.63, I see:

```
$ cargo --version && rustc --version
cargo 1.63.0 (fd9c4297c 2022-07-01)
rustc 1.63.0 (4b91a6ea7 2022-08-08)
$ CARGO_TARGET_DIR=/tmp/sequoia-sq-build cargo build --release --no-default-features --features crypto-botan
...
   Compiling botan-sys v0.10.2
...
error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /home/us/.cargo/registry/src/github.com-1ecc6299db9ec823/botan-sys-0.10.2/src/zfec.rs:10:10
   |
10 |     ) -> core::ffi::c_int;
   |          ^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /home/us/.cargo/registry/src/github.com-1ecc6299db9ec823/botan-sys-0.10.2/src/zfec.rs:20:10
   |
20 |     ) -> core::ffi::c_int;
   |          ^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

```

**Comment by nwalfield** [comment]
That was on commit a99c5ac + `cargo update -p sequoia-openpgp --precise 1.16.0` so `crypto-botan` selected botan3.  Using botan2 works fine.

**Comment by randombit** [comment]
Oops - thanks for the report. In that file we used `core::ffi` directly instead of our indirection type

**Comment by randombit** [comment]
Fixed in 0.10.3 just pushed to `crates.io`

**Comment by nwalfield** [comment]
Thanks for the quick fix. Making a release for us for this fix wasn't necessary. If in the future a release is urgent, I'll be sure to note that. Thanks again!
